hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,4 @@
+RELEASE_TYPE: patch
+
+This patch addresses the issue of hypothesis potentially accessing
+mocked ``time.perf_counter`` during test execution (:issue:`4051`).
```

Comment:
I think this is technically too general a claim, because we also use `perf_counter()` when drawing from a strategy and that can happen inline - e.g. via `st.functions()` or `st.data()`.  But being super-precise here that it addresses _some_ cases doesn't seem like a big deal.

Reply:
A case for centralizing and  replacing `import time` with `from hypothesis.internal import time` perhaps? Later, anyway.